dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/srt/SRTServletRequest.java

```diff
@@ -3343,14 +3349,13 @@ public String getEncodedRequestURI()
             checkRequestObjectInUse();
         }
         // 321485
-        String uri = null;
         if (TraceComponent.isAnyTracingEnabled()&&logger.isLoggable (Level.FINE)) {  //306998.15
```

Comment:
remove these 2 line of logs if you want

Reply:
Done.  I saw that as well when looking at the code, but left it there since I figured it was there for a reason.  I looked at the defect where that code was added and it hasn't changed since then.

Reply:
Some of the methods in this class are lengthy, for example: parseParameter(), will caching the trace gate 

TraceComponent.isAnyTracingEnabled()&&logger.isLoggable (Level.FINE)

be helpeful?

Reply:
It would be minor, but yes it would help a very small amount.